### PR TITLE
Issues: userdashboard: add class to mobile dropdown and update styling in das…

### PR DESCRIPTION
**Pull request by phillimorland**
…hboard_nav to be inline with a+ with small adjustments and fix icon sizes

### CODE REVIEW

File: apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html [hunk 1]
```diff
@@ -135,11 +135,11 @@ <h4>{% trans 'Organisations' %}</h4>
...
                         <a class="btn btn--link text-decoration-none text-muted d-flex align-items-center" href="{% url 'organisation' organisation_slug=organisation.slug %}">
                           {% if organisation.logo %}
-                            <img src="{% thumbnail organisation.logo '0x40' %}" alt="{{ organisation.name }}" />
+                            <img src="{% thumbnail organisation.logo '0x40' %}" alt="{{ organisation.title }}" />
                           {% else %}
-                            <i class="fas fa-home font--lg pt-1" aria-label="{{ organisation.name }} {% trans 'Home' %}"></i>
+                            <i class="fas fa-home font--lg pt-1" aria-label="{{ organisation.title }} {% trans 'Home' %}"></i>
                           {% endif %}
-                            &nbsp {{ organisation.name }}
+                            &nbsp {{ organisation.title }}
```
Comment (khamui) [inline apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html:141]: `organisation.title` is empty, `organisation.name` shows the org name

**Review by khamui — COMMENTED** [review]
cool! Having some comments and questions

File: adhocracy-plus/assets/scss/_base.scss [hunk 2]
```diff
@@ -49,27 +49,28 @@ h6 {
...
 }
 
 h1 {
-    font-size: $font-size-xxxl;
+    font-size: $font-size-xxl;
 }
 
 h2 {
-    font-size: $font-size-xxl;
+    font-size: $font-size-lg;
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply adhocracy-plus/assets/scss/_base.scss:65]: yep difference is capitalising, we don't have a styleguide for kosmo fonts ect. like we do in a+ so it's a kind of guess, in general all our fonts were too big though compared to designs so made some assumptions

File: apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html (hunk 1 above)
Comment (phillimorland) [reply apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html:141]: have you given your oganisation a title? the name comes from the django admin and is normally not as human readable, the title comes from the organisation settings in the dashboard

Comment (khamui) [reply apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html:141]: i see, my bad

File: apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html [hunk 3]
```diff
@@ -99,14 +99,14 @@ <h4>{% trans 'Initiator Organisations' %}</h4>
...
-                        <div class="dropdown"
+                        <div class="userdashboard__dropdown dropdown"
                         data-bs-toggle="collapse"
                         href="#userDashboardCollapseOrgInt"
                         aria-expanded="true"
                         aria-controls="#userDashboardCollapseOrgInt"
                         data-bs-target="#userDashboardCollapseOrgInt">
                           {# Translators: kosmo #}
-                          <h4>{% trans 'Initiator Organisations' %}</h4>
+                          <h2>{% trans 'Initiator Organisations' %}</h2>
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html:108]: actually made it 20 as 16 looked really too small

**Comment by khamui** [comment]
very nice!

**Pull request merged**